src/test/java/jpa/PublisherTest.java

```diff
@@ -0,0 +1,44 @@
+package jpa;
```

Comment:
Update branch as this file is also in the master branch, so no conflicts will occur when pushing.